<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_management</class>
 <widget class="QDialog" name="User_management">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USER MANAGEMENT UTILITY</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>OPTIONS</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>152</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QRadioButton" name="user_passwd_radioButton">
       <property name="text">
        <string>Change password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="usermod_radioButton">
       <property name="text">
        <string>Rename a user</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="userdel_radioButton">
       <property name="text">
        <string>Remove a user</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="useradd_radioButton">
       <property name="text">
        <string>Create a user</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="user_shell_radioButton">
       <property name="text">
        <string>Change login shell</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="list_users_radioButton">
       <property name="text">
        <string>List all users</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter username :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="user_proceed_pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PROCEED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user_exit_pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
